### ISSUE TITLE
liveness detect and anti spoofing?

### ISSUE THREAD

**Issue by neonleo** [exadel-inc/CompreFace#923]
does the face detection engine support liveness check so it can ensure the face is real person .

**Comment by pospielov** [comment]
No, CompreFace uses 2d images, and there is no way to create reliable liveness detection from 2d images.
There are two ways:
1. Make liveness detection from video
2. Make liveness detection from 3d images

CompreFace doesn't support videos or 3d images.
But you can implement your own liveness detection based on video and CompreFace

**Comment by pigping88** [comment]
Can you describe in detail how to realize your own real-time detection based on video and CompreFace
